components/button/button-subtle.js

```diff
@@ -72,36 +72,31 @@ class ButtonSubtle extends ButtonMixin(RtlMixin(LitElement)) {
 					background-color: transparent;
 					border-color: transparent;
 					font-family: inherit;
-					padding: 0 var(--d2l-button-subtle-padding-inline-end) 0 var(--d2l-button-subtle-padding-inline-start);
+					padding-bottom: 0;
+					padding-inline-start: var(--d2l-button-subtle-padding-inline-start);
+					padding-inline-end: var(--d2l-button-subtle-padding-inline-end);
+					padding-top: 0;
```

Comment:
Might as well go all-in!

```suggestion
					padding-block-end: 0;
					padding-block-start: 0;
					padding-inline-start: var(--d2l-button-subtle-padding-inline-start);
					padding-inline-end: var(--d2l-button-subtle-padding-inline-end);
```